src/Redux/Book/Book.js

```diff
@@ -0,0 +1,28 @@
+const ADDBOOK = './bookStore/Book/ADDBOOK';
+const REMOVEBOOK = './bookStore/Book/REMOVEBOOK';
+
+const reduce = [];
+const bookReducer = (state = reduce, action) => {
+  switch (action.type) {
+    case ADDBOOK:
+      return [...state,
+        action.book,
+      ];
+    case REMOVEBOOK:
+      return state.filter((book) => book.id !== action.book.id);
+    default:
+      return state;
+  }
+};
+
+export default bookReducer;
+
+export const addactionCreator = (book) => ({
+  type: ADDBOOK,
+  book,
+});
+
+export const removeactionCreator = (book) => ({
+  type: REMOVEBOOK,
+  book,
+});
```

Comment:
### Your Files and Folder name :red_circle:
![files folder-names (copy)](https://user-images.githubusercontent.com/93039359/186184378-a5a98591-08a9-4292-9f88-7f506001fc3d.png)

### Expected Files and Folder name :green_circle:
```
 /src
        |---/components
        |---/redux
            |--- /books
                    | books.js
            |--- /categories
                    | categories.js
            | cofigureStore.js
        | index.js

```

- [ ] As you can see in your files and folder names image, your files and folder names should start from a small letter because in react only Component files name should start from Capital letter :thinking:

Reply:
Ok, let me fix that

Reply:
Great :+1: